tools/wpt/run.py

```diff
@@ -487,6 +489,19 @@ def setup_wptrunner(venv, prompt=True, install_browser=False, **kwargs):
     setup_cls = product_setup[kwargs["product"]](venv, prompt, sub_product)
     setup_cls.install_requirements()
 
+    affected_revish = kwargs.pop("affected", None)
+    if affected_revish is not None:
+        files_changed, _ = testfiles.files_changed(affected_revish, include_uncommitted=True, include_new=True)
```

Comment:
This is different from the current behaviour of `check_stability`. Namely,

1. `include_uncommitted=True`: why? I think this won't make a difference on CI, but is this an intentional change for local interactive use?
2. `include_new=True`
3. `ignore_changes` isn't taken into into account: this might be a separate concern (the tests *are* affected, but we are unable to run them all because of resource constraints). Besides, IIUC, although the setup in this PR doesn't run stability checks, it'll still attempt to run all testharness.js tests if `testharness.js` is modified, which will still take too long even w/o `--verify`.

Reply:
Yes, I the `=True` args are for local use, although I didn't test them.

testharness.js is excluded by the affected test logic so modifying that actually doesn't cause all those tests to be run. idlharness.js however is not excluded.